cob_monitoring/src/cpu_monitor.py

```diff
@@ -91,15 +92,29 @@ def check_core_temps(self, interval=1):
             # _ vs -
             netdata_module_name_core_temps = ['sensors.coretemp_isa_0000_temperature',
                                               'sensors.coretemp-isa-0000_temperature']
+            error_count = 0
+            netdata_module_name_err = ''
             for name in netdata_module_name_core_temps:
-                netdata_core_temp, error = query_netdata(name, interval)
+                try:
+                    netdata_core_temp, error = query_netdata(name, interval)
```

Comment:
please apply the try-execpt-logic to all usages of `query_netdata` and `query_netdata_info`